### PR TITLE
Utilities add-in

### PR DESCRIPTION
**Pull request by benkates**
Added two helper functions, kebabCase() and dollarFormat(), along with tests.

### CODE REVIEW

File: src/index.test.js [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
+  it("converts a string to kebab-case", () => {
+    expect(kebabCase("Hello World")).toBe("hello-world");
+  });
+});
+
+describe("dollar format test", () => {
+  it("converts a number to a dollar-formatted string", () => {
+    expect(dollarFormat(1000000)).toBe("$1,000,000");
+  });
 });
```
Comment (mitchthorson) [inline src/index.test.js:20]: i'm thinking we organize vitetest tests like these to be close to the modules they relate to, so maybe move these tests to a new test file like `src/lib/utils/index.test.js`. This way we can keep all the relevant stories and tests co-located with the components and modules they relate to.

**Review by mitchthorson — CHANGES_REQUESTED** [review]
This looks great, a couple of small comments left inline. One other thing would be great to include here is adding a line about what's been added to `CHANGELOG.md` file. The way I generally do this, is any features to be included in the next release, I create a new section at the top, and instead of a version number just label it as "Next" like so:
```
## Next

- Add `dollarFormat` utility function
- Add `kebabCase` utility function
```

and then once it is time to release the new version, I just go an update the 'Next' label to match the new version number.

**Comment by benkates** [comment]
@mitchthorson changes incorporated for your review! thank you!

**Review by mitchthorson — APPROVED** [review]

**Comment by mitchthorson** [comment]
LGTM!

**Pull request merged**